docs/pages/eas-update/runtime-versions.mdx

```diff
@@ -100,7 +100,27 @@ It's important to know that this policy does require the developer to manage the
 
 Also, if you select a different native version between Android and iOS, you'll end up with builds and updates with separate runtime versions. When you publish an update, EAS CLI will detect that two updates are needed, and it will create two updates with the required runtime versions and publish them separately.
 
-### `"fingerprintExperimental"` runtime version policy (experimental)
+### `"fingerprint"` runtime version policy
+
+> **warning** The `fingerprint` runtime policy is still in beta as of SDK 51 and not yet considered stable. Use may result in unexpected system behavior.
+
+We provide the `"fingerprint"` runtime version policy for Expo projects that automatically calculates the runtime version for you, including through changes like SDK upgrades or adding custom native code.
+
+```json
+{
+  "expo": {
+    "runtimeVersion": {
+      "policy": "fingerprint"
+    }
+  }
+}
+```
+
+This policy works for both projects with and without custom native code. It works by using the `@expo/fingerprint` package to calculate a hash of your project during builds and updates to determine build-update compatibility (a.k.a. the runtime).
+
+### `"fingerprintExperimental"` runtime version policy
+
+> **warning** `fingerprintExperimental`is deprecated and will be removed in SDK 51.
```

Comment:
```suggestion
> **warning** `fingerprintExperimental`is deprecated and is removed from SDK 51 onwards.
```

Can we rephrase this as per the suggestion above so that we don't have to come back to this doc and update context in the warning to mention that "it has been removed" when we release SDK 51?